src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_DepArrPage.js

```diff
@@ -185,12 +185,13 @@ class CJ4_FMC_DepArrPage {
             fmc.refreshPageCallback = () => fmc.onDepArr();
         };
         //end of CWB EXEC handling
+        modStr = fmc.fpHasChanged ? "MOD[white]" : "ACT[blue]";
 
-        fmc.setTemplate([
-            [originIdent + " DEPARTURES", currentPage.toFixed(0), pageCount.toFixed(0)],
-            ["DEPARTURES" + "[color]blue", "RUNWAYS" + "[color]blue"],
+        fmc._templateRenderer.setTemplateRaw([
+            [" " + modStr + originIdent + " DEPART", currentPage.toFixed(0) + "/" + pageCount.toFixed(0) + " [blue]"],
```

Comment:
I would use template strings here as I think they're a lot easier to mentally parse
```javascript
` ${modStr}${originIdent} DEPART`
```
I've left the comment here but it's a running theme.

Reply:
I wanted to use them, but I think I've tried it once and it didnt work.

But it could be that I used escapes
![image](https://user-images.githubusercontent.com/2995606/94300942-0264d400-ff6a-11ea-9c33-62d534d1a451.png)